EssentialsGeoIP/src/com/earth2me/essentials/geoip/EssentialsGeoIPPlayerListener.java

```diff
@@ -205,4 +214,17 @@ private void downloadDatabase() {
             logger.log(Level.SEVERE, tl("connectionFailed"), ex);
         }
     }
-}
+
+    private boolean checkIfLocal(InetAddress address) {
```

Comment:
So should this only return whether the inetAddress is local to the current machine or whether the address is private as defined in [rfc1918](https://tools.ietf.org/html/rfc1918)? Obviously if the address is within private space it doesn't belong to a country, and is possible in more complex network setups where a bungeecord may forward traffic over a private network and not be setup to forward ip details (or is unable).

Reply:
These changes are not relevant to this PR. They were just an issue because I'm new to working with branches in git and sort of messed up.These changes are not relevant to this PR. They were just an issue because I'm new to working with branches in git and sort of messed up.